stubs/fpdf2/fpdf/enums.pyi

```diff
@@ -89,173 +87,173 @@ class RenderStyle(CoerciveEnum):
     def is_fill(self) -> bool: ...
 
 class TextMode(CoerciveIntEnum):
-    FILL: int
-    STROKE: int
-    FILL_STROKE: int
-    INVISIBLE: int
-    FILL_CLIP: int
-    STROKE_CLIP: int
-    FILL_STROKE_CLIP: int
-    CLIP: int
+    FILL = 0
+    STROKE = 1
+    FILL_STROKE = 2
+    INVISIBLE = 3
+    FILL_CLIP = 4
+    STROKE_CLIP = 5
+    FILL_STROKE_CLIP = 6
+    CLIP = 7
 
 class XPos(CoerciveEnum):
-    LEFT: str
-    RIGHT: str
-    START: str
-    END: str
-    WCONT: str
-    CENTER: str
-    LMARGIN: str
-    RMARGIN: str
+    LEFT = "LEFT"
+    RIGHT = "RIGHT"
+    START = "START"
+    END = "END"
+    WCONT = "WCONT"
+    CENTER = "CENTER"
+    LMARGIN = "LMARGIN"
+    RMARGIN = "RMARGIN"
 
 class YPos(CoerciveEnum):
-    TOP: str
-    LAST: str
-    NEXT: str
-    TMARGIN: str
-    BMARGIN: str
+    TOP = "TOP"
+    LAST = "LAST"
+    NEXT = "NEXT"
+    TMARGIN = "TMARGIN"
+    BMARGIN = "BMARGIN"
 
 class Angle(CoerciveIntEnum):
-    NORTH: int
-    EAST: int
-    SOUTH: int
-    WEST: int
-    NORTHEAST: int
-    SOUTHEAST: int
-    SOUTHWEST: int
-    NORTHWEST: int
+    NORTH = 90
+    EAST = 0
+    SOUTH = 270
+    WEST = 180
+    NORTHEAST = 45
+    SOUTHEAST = 315
+    SOUTHWEST = 225
+    NORTHWEST = 135
 
 class PageLayout(CoerciveEnum):
-    SINGLE_PAGE: Name
-    ONE_COLUMN: Name
-    TWO_COLUMN_LEFT: Name
-    TWO_COLUMN_RIGHT: Name
-    TWO_PAGE_LEFT: Name
-    TWO_PAGE_RIGHT: Name
+    SINGLE_PAGE = "SinglePage"
+    ONE_COLUMN = "OneColumn"
+    TWO_COLUMN_LEFT = "TwoColumnLeft"
+    TWO_COLUMN_RIGHT = "TwoColumnRight"
+    TWO_PAGE_LEFT = "TwoPageLeft"
+    TWO_PAGE_RIGHT = "TwoPageRight"
```

Comment:
At runtime, these look like strings, but they're actually `fpdf.syntax.Name` instances, so this isn't correct:

```pycon
Python 3.11.8 (main, Feb 15 2024, 19:32:18) [Clang 15.0.0 (clang-1500.1.0.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import fpdf.enums
>>> fpdf.enums.PageLayout.SINGLE_PAGE.value
'SinglePage'
>>> type(fpdf.enums.PageLayout.SINGLE_PAGE.value)
<class 'fpdf.syntax.Name'>
```

The same goes for several other enums in this file. I don't know about pyright, but this leads to mypy not correctly understanding the values of these enum members.

On `main`:
```
% MYPYPATH=stubs/fpdf2:stubs/Pillow mypy -c "import fpdf.enums; reveal_type(fpdf.enums.PageLayout.SINGLE_PAGE.value)"
<string>:1: note: Revealed type is "fpdf.syntax.Name"
Success: no issues found in 1 source file
```

With your PR branch:
```
% MYPYPATH=stubs/fpdf2:stubs/Pillow mypy -c "import fpdf.enums; reveal_type(fpdf.enums.PageLayout.SINGLE_PAGE.value)"
<string>:1: note: Revealed type is "Literal['SinglePage']?"
Success: no issues found in 1 source file
```